pkg/apis/apis/v1alpha1/types.go

```diff
@@ -0,0 +1,232 @@
+/*
+Copyright 2022 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	conditionsv1alpha1 "github.com/kcp-dev/kcp/third_party/conditions/apis/conditions/v1alpha1"
+)
+
+// ServiceBinding projects APIs and backing implementations into a Workspace.
```

Comment:
I decided to take the suggestion in the doc and call these `Service*` since we _are_ describing both the objects and the backing implementation.

Reply:
I am not a fan of this. Service is super overloaded. API is precise.

Reply:
From the document used to seed this:

> kind: APIExport # this is really API + behaviour. Better word? ServiceExport?